docs/Tutorials/Private-Network/Create-IBFT-Network.md

```diff
@@ -44,16 +44,18 @@ IBFT-Network/
     ├── data
 ```
 
-### 2. Create Configuration File 
+### 2. Create a configuration file
 
-The configuration file defines the [IBFT 2.0 genesis file](../../HowTo/Configure/Consensus-Protocols/IBFT.md#genesis-file) 
-and the number of node key pairs to generate.    
+The configuration file defines the
+[IBFT 2.0 genesis file](../../HowTo/Configure/Consensus-Protocols/IBFT.md#genesis-file) and the
+number of node key pairs to generate.
 
-The configuration file has 2 subnested JSON nodes. The first is the `genesis` property defining 
-the IBFT 2.0 genesis file except for the `extraData` string. The second is the `blockchain` property 
-defining the number of key pairs to generate. 
-    
-Copy the following configuration file definition to a file called `ibftConfigFile.json` and save it in the `IBFT-Network` directory: 
+The configuration file has two subnested JSON nodes. The first is the `genesis` property defining
+the IBFT 2.0 genesis file, except for the `extraData` string. The second is the `blockchain`
```

Comment:
Maybe explain that extradata is not included in config as it's automatically generated in the resulting genesis file.

Reply:
Done.